### PR TITLE
Pub 1321 fortnightly et list

### PR DESCRIPTION
**Pull request by furnivall**
### JIRA link ###
https://tools.hmcts.net/jira/browse/PUB-1321

### Change description ###
Added ET Fortnightly List.
I've done it in a fairly new way but you would not be wrong to entirely hate it.

**Does this PR introduce a breaking change?** (check one with "x")
My brain is fairly broken after it - does that count?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/views/et-fortnightly-list.njk [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  <details class="govuk-details" data-module="govuk-details" open>
+    <summary class="govuk-details__summary"><span class="govuk-details__summary-text">{{ importantInformation }}</span>
+    </summary>
+    {% set venueEmail = listData['venue']['venueContact']['venueEmail'] %}
+    <div class="govuk-details__text">
+      <p>{{ openJustice1 }}</p>
+      <p>{{ openJustice2 }}</p>
+      <p>{{ openJustice3 }}</p>
+      <p>{{ openJustice4 }}</p>
+      <p>{{ listData['venue']['venueName'] }}, {{ venueEmail }}
```
Comment (ChrisS1512) [inline src/main/views/et-fortnightly-list.njk:31]: The venue data should be in the same paragraph as openJustice4 followed by the :

File: src/test/unit/controllers/EtFortnightlyListController.test.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import sinon from 'sinon';
```
Comment (ChrisS1512) [inline src/test/unit/controllers/EtFortnightlyListController.test.ts:1]: Needs a route unit test file adding

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]
Looks good!

Just a couple of comments + also needs the merge conflict fixing

**Review by junaidiqbalmoj — CHANGES_REQUESTED** [review]
Please add some unit tests for `reshapeEtFortnightlyListData` function :)

File: src/test/unit/controllers/EtFortnightlyListController.test.ts (hunk 2 above)
Comment (furnivall) [reply src/test/unit/controllers/EtFortnightlyListController.test.ts:1]: Done

File: src/main/views/et-fortnightly-list.njk (hunk 1 above)
Comment (furnivall) [reply src/main/views/et-fortnightly-list.njk]: done

**Review by ChrisS1512 — DISMISSED** [review]

**Comment by furnivall** [comment]
> Please add some unit tests for `reshapeEtFortnightlyListData` function :)

done :)

**Review by junaidiqbalmoj — APPROVED** [review]

**Review by ChrisS1512 — DISMISSED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Review by junaidiqbalmoj — APPROVED** [review]

**Pull request merged**